apps/homeless-people-help-map/src/translations/en.ts

```diff
@@ -15,6 +15,7 @@ export default {
     point: "Point",
     districtBorder: "District border",
     drinkingFountain: "Drinking fountain",
+    publicToilet: "Public toilet",
```

Comment:
Use plural here, "Public toilets", for consistency and also for clarity (as it is usually a facility with multiple toilets, and not a single unisex toilet).